.github/workflows/integration-tests-against-emulator.yaml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+name: integration-tests-against-emulator
```

Comment:
FYI: We recently found out that you can also give this a nicer name like "Integration Tests Against the Emulator".

Reply:
Done.